ecr-login/cli/docker-credential-ecr-login/main.go

```diff
@@ -14,13 +14,36 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"os"
+
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/api"
 	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/config"
+	"github.com/awslabs/amazon-ecr-credential-helper/ecr-login/version"
 	log "github.com/cihub/seelog"
 	"github.com/docker/docker-credential-helpers/credentials"
 )
 
+var vrsn bool
+
+const BANNER = `amazon-ecr-credential-helper
+version: %s
+`
+
+func init() {
```

Comment:
Can you move this inside `main()`, just above line 48?  You can either inline it or keep it as a separate named function.  That will still make sure it executes first and avoid potential confusion with a separate `init()` function.

Reply:
Fixed by 527b0da